Dockerfile

```diff
@@ -6,11 +6,12 @@ EXPOSE 8090
 
 ADD build/libs/mxisd.jar /mxisd.jar
 ADD src/docker/start.sh /start.sh
-RUN mkdir -p /var/mxisd
+RUN apk update && apk add bash && rm -rf /var/lib/apk/* /var/cache/apk/* \
+    && mkdir -p /var/mxisd
```

Comment:
Why `mkdir` needed? I made a container without this and it is created, regardless of me binding a host volume to it wiht `-v`.